nise/report.py

```diff
@@ -1020,18 +1031,27 @@ def ocp_create_report(options):  # noqa: C901
             _remove_files(monthly_ros_files)
 
 
-def write_gcp_file(start_date, end_date, data, options):
+def write_gcp_file(start_date, end_date, data, options, file_number=None):
     """Write GCP data to a file."""
     report_prefix = options.get("gcp_report_prefix")
     etag = options.get("gcp_etag") if options.get("gcp_etag") else str(uuid4())
     if not report_prefix:
         invoice_month = start_date.strftime("%Y%m")
         scan_start = start_date.date()
         scan_end = end_date.date()
-        file_name = f"{invoice_month}_{etag}_{scan_start}:{scan_end}.csv"
+
+        base_name = f"{invoice_month}_{etag}_{scan_start}:{scan_end}"
+        if file_number is not None:
+            file_name = f"{base_name}_{file_number:04d}.csv"
+        else:
+            file_name = f"{base_name}.csv"
     else:
-        file_name = report_prefix + ".csv"
-    local_file_path = f"{os.getcwd()}/{file_name}"
+        if file_number is not None:
+            file_name = f"{report_prefix}_{file_number:04d}.csv"
+        else:
+            file_name = report_prefix + ".csv"
```

Comment:
Same question. Let's just remove the if/else and require a file_number to be passed in here.